lf-queue/atomics.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdatomic.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define ATOMIC_SET atomic_flag_test_and_set

```

Comment:
Check if both `ATOMIC_SET` and `ATOMIC_RELEASE` are used or not.

Reply:
Both ATOMIC_SET and ATOMIC_RELEASE are not used.

Reply:
Then, remove unused code. Always keep the code clean and elegant.